bots/tests-scan

```diff
@@ -60,14 +60,19 @@ EXTERNAL_PROJECTS = {
     ],
     'cockpit-project/cockpit-ostree': [
         'cockpit/fedora-atomic',
-        'cockpit/rhel-atomic',
         'cockpit/continuous-atomic',
     ],
     'cockpit-project/cockpit-podman': [
         'cockpit/fedora-28',
     ],
 }
 
```

Comment:
Maybe add a comment here

"# Test context for projects that use Cockpit's non-public images"